### ISSUE TITLE
Can't get Fermata to work on Pixel 6a after android 14 update. Pls help

### ISSUE THREAD

**Issue by tarasov503** [AndreyPavlenko/Fermata#306]
Was working great until the update. Unfortunately, I haven't been able to get it working ever since. I tried deleting cache from Android auto, google Play store, play services, as well as uninstalling / reinstalling the apps. As a last resort I even did a factory reset to delete the update, still haven't had any luck. Not sure what im doing wrong. Have been stuck using my galaxy s21, but would really like it if I could get it working on my pixel.. the pixel connected to my android auto dongle so much faster in my truck..
Any suggestions?

**Comment by aamaamilan** [comment]
same issue here with P6P, with andeoid 14 I couldn't find any solution, it's only rooting can solve  this. also with the P7P it's not working.
Hope Andrey will help us to solve this.

**Comment by CantStandYa8** [comment]
Any luck with getting fermata auto to work on Google Pixel 7 Android 14 without rooting the device?  I've tried most of the troubleshooting methods listed in the forum. I had it working no problem up until yesterday when the fermata app updated. Any and all help greatly appreciated

**Comment by divebombdom** [comment]
Having this issue on pixel 8 pro as well.

**Comment by CantStandYa8** [comment]
I've tried most everything across the different Github forums. I cannot get Kinginstaller to install the app "from Google play store".

I had success on Android 13 when changing the package installer. However in Android 14, the installation of the ApkMirror android 14 package installer does not complete and dissapears. Almost like its blocked

I thunk I've exhausted most every solution I've been able to find. Nuclear option being flash back to Android 13

**Comment by ShelbyGT35** [comment]
Please work on the functions for Android 14!!!
Thank you.

**Comment by Oops4me** [comment]
I m big fan of Fermata auto. But @AndreyPavlenko cannot have a fix for it as the OS detects that the app is not from Google play store and rejects the app.

I m sure he would have worked on the fix.

But sadly this would be the end of fermata auto. :(

**Comment by ShelbyGT35** [comment]
I don't think so. Google checked foreign apps for necessary requirements already in the past.
Insofar I'm quite sure he will fix the problem as soon as possible.

**Comment by Qnehub** [comment]
I bought a AAwireless device last week and just recieved it today. Fermata Auto is back to working again on my S21 Ultra after it got killed following the update to Android 14/One UI 6. Just need to turn on developer mode in the AAwireless options to get it back.

I'm happy with the wireless connectivity aside from Fermata Auto since my car doesn't naturally support wireless Android Auto, so I'm pretty happy with the solution of buying the Awireless device.
